src/geohash_helper.c

```diff
@@ -233,3 +241,10 @@ int geohashGetDistanceIfInRadiusWGS84(double x1, double y1, double x2,
                                       double *distance) {
     return geohashGetDistanceIfInRadius(x1, y1, x2, y2, radius, distance);
 }
+
+int geohashGetDistanceIfInRectangle(double *bounds, double x1, double y1,
+                                    double x2, double y2, double *distance) {
+    if (x2 < bounds[0] || x2 > bounds[2] || y2 < bounds[1] || y2 > bounds[3]) return 0;
```

Comment:
is this what you meant?
```suggestion
    if (x2 < bounds[0] || x1 > bounds[2] || y2 < bounds[1] || y1 > bounds[3]) return 0;
```

Reply:
ohh, maybe i'm wrong. i thought x1,y1,x2,y2 define a second box (to be matched against `bounds`).
but that indeed doesn't make sense.
from your comment i understand that x1,y1 define the center of the box (or circle in the radius counterpart).
and that x2,y2 define the point to be searched.
please rename these and / or add comments to make it clear (in both radius and rectangle variants)